esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -851,28 +949,44 @@ EsdlBindingImpl::EsdlBindingImpl()
 {
 }
 
-EsdlBindingImpl::EsdlBindingImpl(IPropertyTree* cfg,
-                                 const char *binding,
-                                 const char *process) : CHttpSoapBinding(cfg, binding, process)
+EsdlBindingImpl::EsdlBindingImpl(IPropertyTree* cfg, const char *binding,  const char *process) : CHttpSoapBinding(cfg, binding, process)
 {
     m_bindingName.set(binding);
     m_processName.set(process);
 
     try
     {
-        DBGLOG("ESDL Binding %s is subscribing to all /ESDL/Bindings/Binding dali changes", binding);
-        //Since it seems we cannot subscribe to a specific /ESDL/Bindings/Binding, all bindings have to subscribe to the /ESDL/Bindings branch
-        m_pBindingSubscription.clear();
-        m_pBindingSubscription.setown( new CESDLBindingSubscription(this) );
+        char currentDirectory[_MAX_DIR];
```

Comment:
Borrowed concept of storing state files in currentdirectory from legacy code, I'm not crazy about this approach. Do we have an adequate location for this type of data?